adam-core/src/main/scala/org/bdgenomics/adam/rdd/features/FeatureRDD.scala

```diff
@@ -95,10 +102,117 @@ object FeatureRDD {
 
     FeatureRDD(rdd, sd)
   }
+
+  /**
+   * @param feature Feature to convert to GTF format.
+   * @return Returns this feature as a GTF line.
+   */
+  private[features] def toGtf(feature: Feature): String = {
+    def escape(entry: (Any, Any)): String = {
+      entry._1 + " \"" + entry._2 + "\""
+    }
+
+    val seqname = feature.getContigName
+    val source = Option(feature.getSource).getOrElse(".")
+    val featureType = Option(feature.getFeatureType).getOrElse(".")
+    val start = feature.getStart + 1 // GTF/GFF ranges are 1-based
+    val end = feature.getEnd // GTF/GFF ranges are closed
+    val score = Option(feature.getScore).getOrElse(".")
+    val strand = Features.asString(feature.getStrand)
+    val frame = Option(feature.getFrame).getOrElse(".")
+    val attributes = Features.gatherAttributes(feature).map(escape).mkString("; ")
+    List(seqname, source, featureType, start, end, score, strand, frame, attributes).mkString("\t")
+  }
+
+  /**
+   * @param feature Feature to write in IntervalList format.
+   * @return Feature as a one line interval list string.
+   */
+  private[rdd] def toInterval(feature: Feature): String = {
+    val sequenceName = feature.getContigName
+    val start = feature.getStart + 1 // IntervalList ranges are 1-based
+    val end = feature.getEnd // IntervalList ranges are closed
+    val strand = Features.asString(feature.getStrand, emptyUnknown = false)
+    val attributes = Features.gatherAttributes(feature)
```

Comment:
Just wanted to get back to this, the reason I've shoved records in here is that this is the format we got in the interval list file that described the exome panel. I don't know what the "best" approach is, but I don't think interval list is a rigorously specified format, so we're somewhat making things up as we go here.